x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots/get_element_position_data.ts

```diff
@@ -66,8 +68,10 @@ export const getElementPositionAndAttributes = async (
       );
     }
   } catch (err) {
+    apm.captureError(err);
     elementsPositionAndAttributes = null;
   }
 
+  if (apmSpan) apmSpan.end();
```

Comment:
I think the null check (`if (apmSpan)`) is unnecessary here?  (same comment in other files)

Reply:
The `startSpan` function returns `Span` or `null`. My understanding is `null` is returned if APM is disabled. 

I thought about trying some ways to abstract that with helper functions, but in the end, it's not very much boilerplate. Also, the typings for APM objects aren't available to import, so helper functions are a little difficult with APM.